test/OpenTelemetry.Extensions.Hosting.Tests/OpenTelemetryServicesExtensionsTests.cs

```diff
@@ -450,9 +451,48 @@ public void AddOpenTelemetry_WithLogging_NestedResolutionUsingConfigureTest()
         Assert.True(innerTestExecuted);
     }
 
+    [Fact]
+    public async Task AddOpenTelemetry_HostedServiceOrder_DoesNotMatter()
+    {
+        var exportedItems = new List<Activity>();
+
+        var builder = new HostBuilder().ConfigureServices(services =>
+        {
+            services.AddHostedService<TestHostedService>();
```

Comment:
without adding our hostedservice at the very beginning, Activity from this TestHostedService would not have been collected, right? 
Could you add a changelog entry as well and perhaps modify the PR description to show the exact scenario this is fixing?

Reply:
Correct, this test failed before this change.